packages/requisitions/src/RequestRequisition/DetailView/RequestLineEdit/ItemCharts/StockDistribution.tsx

```diff
@@ -0,0 +1,185 @@
+import React from 'react';
+import {
+  Box,
+  ItemStatsNode,
+  Tooltip,
+  Typography,
+  useTranslation,
+} from '@openmsupply-client/common';
+import { useRequestFields } from '../../../api';
+
+export interface StockDistributionProps {
+  itemStats?: ItemStatsNode;
+  suggestedQuantity?: number;
+}
+
+const MIN_FLEX_BASIS_TO_SHOW_LABEL = 10;
+const MIN_FLEX_BASIS_TO_SHOW_VALUE = 5;
+const MIN_MC_WIDTH_TO_SHOW_TEXT = 5;
+
+const Divider = () => (
```

Comment:
I have a bunch of components in this file - they are all specific to the StockDistribution component, so I thought it best to keep them all in the one file. Might be too many though?

Reply:
Yup, they're small and it was good to read and understand